exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -49,10 +49,10 @@ fun <T : Comparable<T>, S : T?> ExpressionWithColumnType<in S>.avg(scale: Int =
 fun <T : Any?> ExpressionWithColumnType<T>.sum(): Sum<T> = Sum(this, this.columnType)
 
 /** Returns the number of input rows for which the value of this expression is not null. */
-fun ExpressionWithColumnType<*>.count(): Count = Count(this)
+fun ExpressionWithColumnType<*>.count(): Count = Count(this, false, this.columnType)
```

Comment:
Why count is changed? Count could not return double or other non integer value. And what value should be returned when you call it on VarChar column? 
Please revert that change.

Reply:
Completely agree. Sorry, I misunderstood the background. I'll revert it.

Reply:
fixed in 
94012b8